test/libsolidity/SolidityScanner.cpp

```diff
@@ -361,7 +361,7 @@ BOOST_AUTO_TEST_CASE(multiline_documentation_comments_parsed)
 	BOOST_CHECK_EQUAL(scanner.next(), Token::Identifier);
 	BOOST_CHECK_EQUAL(scanner.next(), Token::Identifier);
 	BOOST_CHECK_EQUAL(scanner.next(), Token::EOS);
-	BOOST_CHECK_EQUAL(scanner.currentCommentLiteral(), "Send $(value / 1000) chocolates to the user");
+	BOOST_CHECK_EQUAL(scanner.currentCommentLiteral(), " Send $(value / 1000) chocolates to the user");
```

Comment:
The additional space makes the behavior between singleline and multiline docstring comments consistent. For example, the above code would be similar to 
```
///
/// Send $(value / 1000) chocolates to the user
```

The singleline docstring will parse the space after the second `///`